Arrowgene.Ddon.Shared/Model/Craft/NpcActionType.cs

```diff
@@ -0,0 +1,11 @@
+namespace Arrowgene.Ddon.Shared.Model
+{
+    public enum NpcActionType : byte
+    {
+        NpcActionNone = 0,
+        NpcActionStithy = 1,
```

Comment:
Well it could also be an archaic expression for "forge" or "anvil" (had to look that one up..). It's from the debug symbols, but I can change it to smithy which is still in the same semantic category as the archaic expression.

Btw., the whole NpcAction thing seems completely unused. There is exactly one NPC handling all kinds of crafting. Not sure what this is supposed to express.